From 8bcac7a86fa7b9e44ddf3ab166f84c8e4742d0e5 Mon Sep 17 00:00:00 2001
From: Matthias Clasen <mclasen@redhat.com>
Date: Fri, 17 Jul 2015 16:14:59 -0400
Subject: [PATCH 4/7] inspector: Annotate a call whose return value we don't
 need

The way this code is written, we know that there is an item in
the hash table and the iter_next call will give it to us, so
no need to check the return value. Annotate the call to tell
coverity.
---
 gtk/inspector/gestures.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gtk/inspector/gestures.c b/gtk/inspector/gestures.c
index d9d727a..9463c12 100644
--- a/gtk/inspector/gestures.c
+++ b/gtk/inspector/gestures.c
@@ -200,7 +200,7 @@ gtk_inspector_gestures_set_object (GtkInspectorGestures *sl,
           gpointer key, value;
           GtkGesture *gesture;
           g_hash_table_iter_init (&iter, hash);
-          g_hash_table_iter_next (&iter, &key, &value);
+          (void)g_hash_table_iter_next (&iter, &key, &value);
           gesture = key;
           add_gesture_group (sl, object, gesture, hash);
         }
-- 
2.4.5

